src/pages/settings/Profile/PersonalDetails/StateSelectionPage.tsx

```diff
@@ -91,13 +91,12 @@ function StateSelectionPage() {
                 shouldShowBackButton
                 onBackButtonPress={() => {
                     const backTo = params?.backTo ?? '';
-                    let backToRoute = '';
+                    let backToRoute = '' as Route;
```

Comment:
Suggestion:
```ts
                    let backToRoute: Route | undefined;

                    if (backTo) {
                        backToRoute = appendParam(backTo, 'state', currentState ?? '');
                    }

                    Navigation.goBack(backToRoute);
```

I think we should fix `appendParam` so that it returns a `Route` type. This way we can avoid assertion in this file